cpu/riscv_common/irq_arch.c

```diff
@@ -129,7 +147,7 @@ void handle_trap(uint32_t mcause)
 
 /* Marking this as interrupt to ensure an mret at the end, provided by the
  * compiler. Aligned to 4-byte boundary as per RISC-V spec  */
```

Comment:
comment needs to be updated

Reply:
Nah, the alignment modification was not necessary, reverted that so the comment is valid again :)